simapp/helpers/test_helpers.go

```diff
@@ -0,0 +1,43 @@
+package helpers
+
+import (
+	"math/rand"
+
+	"github.com/tendermint/tendermint/crypto"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+)
+
+// GenTx generates a signed mock transaction.
+func GenTx(msgs []sdk.Msg, accnums []uint64, seq []uint64, priv ...crypto.PrivKey) auth.StdTx {
```

Comment:
ideally, we should move all the contents from `simapp/test_helpers.go` to this file and update the keepers that use it, although that's out of scope from this PR.